ldelf/start_rv64.S

```diff
@@ -0,0 +1,62 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2022 ESWIN Corp. or its affiliates.
+ */
+
+
+#include <asm.S>
+#include <elf_common.h>
+
+/*
+ * _start() - Entry of ldelf
+ *
+ * See include/ldelf.h for details on TEE Core interaction.
+ *
+ * void start(struct ldelf_arg *arg);
+ */
+FUNC _ldelf_start , :
+	/*
+	 * First ldelf needs to be relocated. The binary is compiled to
+	 * contain only a minimal number of R_RISCV_RELATIVE relocations in
+	 * read/write memory, leaving read-only and executeble memory
+	 * untouched.
+	 */
+	lla a1, reloc_end_rel
```

Comment:
Please add a tab character between the instruction and its arguments. This comment applies to all the instructions in this file.